alacritty.yml

```diff
@@ -21,6 +21,14 @@ window:
     columns: 0
     lines: 0
 
+  # Window geometry (changes require restart)
+  #
+  # Specified in number of pixels.
+  # If both are `0`, this setting is ignored.
+  geometry:
+    x: 0
+    y: 0
```

Comment:
This should *not* be ignored when both are `0`, since `0x0` is not the default window position.

I think the best option here would be to just have `geometry` commented out by default. While that's usually a bad choice because there's no way to explicitly disable it, I think it might be better than something like `geometry: Default`.

Reply:
Is everything in `config.rs` required to have a partner in `alacritty.yml`? If not, what's your opinion on not letting this be configurable by the config file at all? Frankly, it makes everything a whole lot easier to implement, and I can't think of a time when you'd want to set it from there.

Reply:
> Is everything in config.rs required to have a partner in alacritty.yml?

Yes.

> If not, what's your opinion on not letting this be configurable by the config file at all?

Not an option at all.

> it makes everything a whole lot easier to implement

I don't think that's the right thing to focus on. Even with a configuration option this is a fairly straight-forward feature. Skimping out on it and putting the load of working around it on the user is not the right move here.